tests/src/test/java/alluxio/hadoop/HdfsFileInputStreamIntegrationTest.java

```diff
@@ -67,11 +74,12 @@ public final void after() throws Exception {
 
   @Before
   public final void before() throws IOException, AlluxioException {
-    AlluxioURI file1 = new AlluxioURI("/testFile1");
-    mInMemInputStream = new HdfsFileInputStream(file1, new Configuration(), BUFFER_SIZE, null);
+    mFile1 = new AlluxioURI("/testFile1");
+    mInMemInputStream = new HdfsFileInputStream(mFile1, new Configuration(), BUFFER_SIZE, null);
 
-    AlluxioURI file2 = new AlluxioURI("/testFile2");
-    mUfsInputStream = new HdfsFileInputStream(file2, new Configuration(), BUFFER_SIZE, null);
+    ClientContext.getConf().set(Constants.USER_FILE_READ_TYPE_DEFAULT, "NO_CACHE");
```

Comment:
Actually, I was intended to apply `NO_CACHE` to all underlying tests. Because original unit test **always** uses the default read type `CACHE_PROMOTE ` on underlying input stream, which could become problematic if several consecutive reads included in the same test. I'll find a better way to handle this situation.